contents/security/crossSiteScripting/crossSiteScripting.md

```diff
@@ -76,35 +99,102 @@ However, that also means that hackers are able to also include scripts in their
 
 `https://www.ecard.com/view-ecard.php?content=Happy%20Holidays<script>...</script>`
 
-## Preventing Cross Site Scripting
+## Well-Known XSS Incidents
+
+**MySpace Worm**
+
+In 2005, MySpace allowed users to customise their profiles using HTML code. This allowed for diversity of profiles but also allowed Samy Kamkar to find an XSS vulnerability. 
```

Comment:
.. allowed a user named Samy Kamkar to find an XSS vulnerability.

Reply:
Changed